src/views/registration/attendees/content.php

```diff
@@ -18,15 +22,14 @@
 }
 
 $storage = new Tribe__Tickets_Plus__Meta__Storage();
-$meta    = tribe( 'tickets-plus.main' )->meta();
+/** @var Tribe__Tickets_Plus__Meta $meta */
+$meta    = tribe( 'tickets-plus.meta' );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: facbd0af6e1765ad29530cd68943c203</sub>